src/clj/kamera/core.clj

```diff
@@ -265,7 +270,8 @@
                                                    ;; see element-exists?
                          :assert?              true ;; runs a clojure.test assert on the expected/actual when true, makes no assertions when false
                          :resize-to-contents   {:height? true
-                                                :width? false}}
+                                                :width? false
+                                                :dom-selector "body"}}
```

Comment:
@oliyh `:resize-to-contents` is a map, thus the inner pair of braces and the requirement for a `deep-merge` below.